### ISSUE TITLE
[BUG] Typings for 2.6.14 are broken

### ISSUE THREAD

**Issue by arutkowski00** [framer/motion#768]
TypeScript type definitions from `dist/` cannot be built with `tsc` as they contain invalid imports.

```
export declare const m: import("./types").HTMLMotionComponents & import("./types").SVGMotionComponents & {
                               ~~~~~~~~~

node_modules/framer-motion/dist/framer-motion.d.ts:2304:73 - error TS2307: Cannot find module './types' or its corresponding type declarations.

export declare const m: import("./types").HTMLMotionComponents & import("./types").SVGMotionComponents & {
                                                                        ~~~~~~~~~

node_modules/framer-motion/dist/framer-motion.d.ts:2305:138 - error TS2307: Cannot find module '.' or its corresponding type declarations.

    custom: <Props>(Component: string | import("react").ComponentClass<Props, any> | import("react").FunctionComponent<Props>) => import(".").CustomDomComponent<Props>;
                                                                                                                                         ~~~

```

There's no CodeSandbox as it does not provide any typings errors. Just look at `framer-motion.d.ts` in `framer-motion@2.6.14`, lines `2300-2305`:

```
/**
 * @public
 */
export declare const m: import("./types").HTMLMotionComponents & import("./types").SVGMotionComponents & {
    custom: <Props>(Component: string | import("react").ComponentClass<Props, any> | import("react").FunctionComponent<Props>) => import(".").CustomDomComponent<Props>;
};
```

The code contains invalid imports that were not present in previous versions.

**Steps to reproduce the behavior:**
1. Create a project with `framer-motion@2.6.14`
2. Import something from this package
3. Try to build the app with `skipLibCheck: false` in `tsconfig.json`

**Expected behavior**

Typings should be compilable via `tsc`

**Environment details**

TypeScript v3.9.5
Node v14.5.0

**Comment by InventingWithMonster** [comment]
@arutkowski00 Can you check `2.6.15-rc.0` to see if this fixes it for you?

**Comment by InventingWithMonster** [comment]
@arutkowski00 Can you confirm this has been fixed in `2.6.15`?

**Comment by InventingWithMonster** [comment]
I'm going to close this for now but feel free to reopen if you run into any more troubles with this.
